src/frontend/js/components/DashBoard/Views/CourseList.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
As we talked about, remove this component then return directly the JSX into the routes object like it did for preferences view.

In this way, you can delete the `Views` folder that is currently useless.